src/proposal.ts

```diff
@@ -17,11 +18,12 @@ export enum ProposalOutcome {
 }
 
 export enum ProposalStage {
-  Open,
-  Boosted,
-  QuietEndingPeriod,
+  ExpiredInQueue,
+  Executed,
   Queued,
-  Resolved
+  PreBoosted,
```

Comment:
so there isn't another stage for Pending Boost, its still PreBoosted but just has enough gens to be boosted after pending time period passes?